src/Utils.ts

```diff
@@ -0,0 +1,23 @@
+import * as core from '@actions/core';
+
+export class Utils {
+    static validateSettings(customSettings: string, maskInputs?: string) {
+        try {
+            var customParsedSettings = JSON.parse(customSettings);
+            if (!!maskInputs && maskInputs !== "false") {
```

Comment:
this should be an || instead of &&

Reply:
if input does not exist or if it exists, it should not be false.

Reply:
```
if (!maskInputs || maskInputs !== "false")
```

Reply:
i had added !!maskInputs to check if its not undefined since i had it as an optional argument.
in master general-settings is not an optional argument but in releases/v1 it is, that is why i added optional argument.
since we need to merge code changes from master into releases/v2, i am removing optional argument and removing the !!maskInputs condition as well